pkg/k8sutils/k8sutils.go

```diff
@@ -154,3 +158,17 @@ func CreateCRD(resource apiextensions.CustomResource) error {
 	}
 	return nil
 }
+
+// GetImageRegistryFromStorkDeployment - extract image registry and image registry secret from stork deployment spec
+func GetImageRegistryFromStorkDeployment() (string, string, error) {
```

Comment:
lets pass in name/namespace as well so that we can use this call for other importing libs where stork is not installed in default ns as well as get any deployment image details

Reply:
Fixed. Made the API to generic by taking deployment name and namespace.